### ISSUE TITLE
Media section on the homepage

### ISSUE THREAD

**Issue by AndySok** [stuyspec/stuyspec.com#325]
A fairly straight-forward change to implement. As requested by EICs, the bottom right column of the website is going to have a media section as opposed to a news section, to make sure media spreads are noticed by the student body. Links to [API].

**Comment by TheCheesyOlympia** [comment]
The current news section on the bottom-left of the title page is redundant and somewhat cluttered, and as such, is a prime candidate for replacement. 

First of all, as mentioned by the issue, in order to encourage the reading of media spreads, the left column should be dedicated instead to displaying information pertaining to media instead.

**An additional note**: The current format of the left column is rather cluttered, as there is no caption denoting it as the news section. Instead, every article has a news byline on the top, which is hard to follow, especially when there are multiple articles. As such, the new Media section should have a large header on top, and clear borders to denote its importance, followed by the list of the media spreads.

- Jeffrey Mui (6/5/21)

**Comment by izzyjia** [comment]
The News section on the bottom right column of the website seems out of place since there already is a whole column dedicated to News on the bottom left side. It takes up the whole column, as opposed to Humor, Opinions, Features, and A & E, which have smaller space designations, usually with three departments sharing a column. I think News already has enough representation and the News section on the bottom right could be replaced with a Media Section. However, I'm not sure how many images media could fit into that space, so maybe a collage or array of images could be displayed instead.

There is also empty space in the bottom left corner under the News and Humor columns, which could be utilized to add more writing sections, perhaps for the Science Department. Also, the headers for each department look a bit small and could have their font sizes increased, or a different shade of color because it's hard to distinguish them from the titles of articles. 

Additionally, a small detail I noticed for the Stuyvesant offers AP Art article, the title precedes the News byline, which is different from the rest of the article formats. I think it would be nice to put the News byline could be placed on top of the article name to maintain consistency. 
- Isabella Jia (6/7/21)
